### ISSUE TITLE
text selector on FXer

### ISSUE THREAD

**Issue by matanbanin** [renatoathaydes/Automaton#31]
Hi, as the last one who opened an issue we also enjoy your framework very much, and use it to test our apps. Respect for the awsome work.

But when i try to use the fxer.getAt("text:someText") selector i get an exception of MissingMethodException in TableHeaderRow.getText()

Thank you! and please continue on the good work.

**Comment by renatoathaydes** [comment]
Thank you for the compliments!! I appreciate it.

Please post the whole stack-trace if possible. That makes is much easier to fix the issue.

**Comment by matanbanin** [comment]
we work on separate lan and it is huge bureaucracy to bring it into here...
can i give you a specific information from it?

**Comment by renatoathaydes** [comment]
What's the first (from top to bottom) class and line from Automaton code which shows up in the stack trace?

For example:
```
MyException at:
a.b.C:10
com.athaydes.automaton.Automaton:54
com.athaydes.automaton.Automaton:23
c.d.E:22
e.f.G:33
```

I just need to know `Automaton:54` from this stack-trace.

If you have any `caused by OtherException` in it, then I would be interested in `OtherException` instead.

**Comment by matanbanin** [comment]
![image](https://cloud.githubusercontent.com/assets/14214701/9989624/d426b4b0-6062-11e5-8985-14b887506126.png)

**Comment by renatoathaydes** [comment]
You seem to be using an older version of Automaton. I cannot find out why this is happening in your tests, but please try to upgrade to the latest version of Automaton (1.2.3 as of today)... The code does check if the method exists before calling it... I would be really surprised if that still happened after you upgraded.

**Comment by renatoathaydes** [comment]
Ok, I wrote a test to select the header of a JavaFX Table, which is what you're trying to do here, and got the same error... Someone thought it was a good idea to have a getText method in this class which requires an argument! I didn't expect that to happen (getText is supposed to be a simple getter)!!

Will fix for the next release, which is maybe a few days ahead...

**Comment by renatoathaydes** [comment]
Problem fixed. Will release the fix soon with version 1.2.4. There's a PR in progress which I want to merge before releasing... shouldn't take more than a couple of days. You can try this fix by checking out the code yourself, if it's important.

**Comment by matanbanin** [comment]
Awosome :) :+1: 
Thank you very much! 
Do you have an assessment of when it will be released?

**Comment by renatoathaydes** [comment]
I should make a release in one week or two... I have been splitting out part of Automaton into a new library... It's done already, but I need to write new docs, test some more, update documentation etc.

**Comment by renatoathaydes** [comment]
Hello @matanbanin I've released version 1.3.0-beta (no 1.2.4 as too many things were changed). Please try it out.
